packages/theme-default/src/components/Overview/index.tsx

```diff
@@ -144,6 +156,12 @@ export function Overview(props: {
     items: (NormalizedSidebarGroup | SidebarItem)[];
   };
 
+  const {
+    apiFilterText = 'Filter',
```

Comment:
I think the config name should be `overviewFooText`, since it is not only used in api page.

Reply:
OK, this is better.